FabNeso.py

```diff
@@ -0,0 +1,58 @@
+# -*- coding: utf-8 -*-
+#
+# This source file is part of the FabSim software toolkit, which is
+# distributed under the BSD 3-Clause license.
+# Please refer to LICENSE for detailed information regarding the licensing.
+#
+# This file contains FabSim definitions specific to FabNeso
+#
+# authors: Duncan Leggat
+
+try:
+    from fabsim.base.fab import *
+except ImportError:
+    from base.fab import *
```

Comment:
Realise this is just taken from `FabDummy` example, but I would avoid `from ... import *` (and in fact Ruff may well disallow this) imports in favour of just importing the specific names you need, so for example something like

```Python
from fabsim.base.fab import (
    add_local_paths,
    execute,
    find_config_file_path,
    job,
    run_ensemble,
    task,
    update_environment,
    with_config, 
)
```
or just use import the relevant module and use qualified names e.g.

```Python
try:
    import fabsim.base.fab as fab
except ImportError:
    import base.fab as fab
```
then use `fab.add_local_paths`, `@fab.task` etc.

Reply:
When I do this, I get new errors from mypy complaining when I redefine the same paths as: 
```python
except ImportError:
    from base.fab import (
        add_local_paths,
    )
``` 
with an error:
```
FabNeso.py:25: error: Name "add_local_paths" already defined (possibly by an import)  [no-redef]
``` 
Did this come up for you if you define them in both the try and except stages?

Reply:
I ended up using the latter 

```Python
try:
    import fabsim.base.fab as fab
except ImportError:
    import base.fab as fab
```

approach then qualifying names with `fab.` to avoid the duplication of names being imported.

Reply:
In the end the issues seemed to come because I'd initially set up the repository in the plugin directory within FabSIM itself - I moved the plugin to another directory elsewhere on my computer and ran it again and the import errors were no longer raised. This also solved the errors I was getting from mypy in the fabsim base, which it no longer raises.